src/igvfd/types/analysis_step.py

```diff
@@ -31,6 +31,8 @@ class AnalysisStep(Item):
         Path('parents', include=['@id', 'title', 'status']),
         Path('workflow', include=['@id', 'accession', 'status']),
         Path('submitted_by', include=['@id', 'title']),
+        Path('analysis_step_versions.software_versions', include=[
+             '@id', 'analsyis_step_versions', 'software_versions', 'name'])
```

Comment:
There's a typo here, although it seems like the embedding isn't broken, but should probably fix.

Reply:
thanks for catching that. fixed